test/fixture/resolve.mjs

```diff
@@ -1,9 +1,9 @@
 import { resolvePath, createResolve, resolveImports } from 'mlly'
 
-import.meta.resolve = createResolve({ url: import.meta.url })
-console.log(await import.meta.resolve('./cjs.mjs'))
+const resolve = createResolve({ url: import.meta.url })
+console.log(await resolve('./cjs.mjs'))
```

Comment:
I see that those test files are not really used for anything right now (which is a shame because it would have caught an issue otherwise) but I've changed this to created a local variable rather than assigning to `import.meta.resolve` as `import.meta.resolve` has its own type already (coming from Node) so even though it's our custom resolver, it has still used the Node type and thus wouldn't check type issues if those would be inconsistent.